certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -23,6 +23,34 @@
   ],
   "paths": {
     "/webhooks/schema/{listID}": {
+      "get": {
+        "description": "Schema for selected item in action (deprecated)",
+        "summary": "Item schema",
+        "operationId": "GetListSchema",
```

Comment:
@StefanCRT22 The operationId 'GetListSchema' is used multiple times. Operation identifiers must be unique. Please change the operationId for the new action.
Thank you so much.